scripts/merged-prs/download.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ "$#" -ne 3 ]; then
+  echo "Usage: $0 repository start_date end_date" >&2
+  echo "Example: $0 input-output-hk/cardano-node newhoggy 2023-02-22 '2023-02-22 +14 days'" >&2
+  exit 1
+fi
+
+if [[ "$(uname -s)" == "Darwin" ]]; then
+  date_cmd=gdate
+else
+  date_cmd=date
+fi
+
+repository="$1"
+start_date="$("$date_cmd" -u -d "$2" +"%Y-%m-%dT%H:%M:%SZ")"
+end_date="$("$date_cmd" -u -d "$3" +"%Y-%m-%dT%H:%M:%SZ")"
+
+mkdir -p merged-prs
+
+gh pr list --repo "$repository" \
+  -L 1000 \
```

Comment:
Will 1000 be enough?

Reply:
Now get's the total number of PRs ever created.